### ISSUE TITLE
request, change order of modes in [openpanel]

### ISSUE THREAD

**Issue by porres** [pure-data/pure-data#816]
openpanel now has two extra modes since https://github.com/pure-data/pure-data/commit/713669388e7f71f8631e9d7335ee7b6bbc55c8ee , see https://github.com/pure-data/pure-data/pull/592

but I think it makes better sense if mode 1 is 'multiple files' and 'mode 2' is 'directory.

This is because mode 0 is also getting a file, so 0 and 1 would be more closely related, then mode 2 would be for directory.

It seems less intuitive that you have mode 0 for a file, then you kinda change category to opening a directory, but then you're back to a file mode with mode 2... I hope I'm making sense here

**Comment by umlaeute** [comment]
hmm. using a symbol for the "mode" (eg "file", "directory", " files") might make sense, but i would rather not spend brainpower on what "seems less intuitive" about the ordering of numeric modes.

**Comment by porres** [comment]
>  i would rather not spend brainpower on what "seems less intuitive" about the ordering of numeric modes.

Yeah, I know I'm probably spending brainpower for nothing really important, but it annoy me a little so I went for it

**Comment by reduzent** [comment]
I already have been  using those modes. Changing them would break my stuff.

(Not that I worry much about breaking my stuff, it'd be an easy fix. But once stuff is released, I wouldn't recommend to change them too light-heartedly.)

**Comment by reduzent** [comment]
BTW: Those modes are non-intuitive enough so that they have to be looked up anyway. I don't think it makes much sense to "optimize" the order.

**Comment by porres** [comment]
> once stuff is released, I wouldn't recommend to change them

Of course! Me too and I did mention that in the list. Anyway, I had hopes this would have been merged "immediately" after the other PR (but things slowed down). That would avoid this discussion wether it's really intuitive or not or worth the hassle at this point... I had the fear that if it took too long, it could lead to people laughing at how picky I am on minor design details :/
